evrApp/src/evr/evr.h

```diff
@@ -28,6 +28,22 @@ class Input;
 class CML;
 class DelayModuleEvr;
 
+/* PLL Bandwidth Select (see Silicon Labs Si5317 datasheet)
+ *  000 - Si5317, BW setting HM (lowest loop bandwidth)
+ *  001 - Si5317, BW setting HL
+ *  010 - Si5317, BW setting MH
+ *  011 - Si5317, BW setting MM
+ *  100 - Si5317, BW setting ML (highest loop bandwidth)
+ */
+enum PLLBandwidth {
```

Comment:
Something as hardware specific should probably live in `drvem.h` or even `evrRegMap.h`.

Reply:
This part of the code is common to both the EVR and EVM. Do you have any suggestions to consolidate it?

Reply:
This is what `mrmShared/src/` is for.

Actually, I don't that this enum is being used, so maybe it can simply be removed?

Reply:
Actually, it is an artifact of a previous modification. I was using it as a datatype for the functions but have switched to using `epicsUInt16`. I removed the `enum` definition and added a `#define PLLBandwidth_MAX` to both the `evrRegMap.h` and `evgRegMap.h` files.